updates/6/updates.sql

```diff
@@ -233,4 +233,76 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xF962B098Ecc43
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("STEP", "Y", "Step Finance", "STEP", 1619335055, NULL, "step-finance", "STEP", "STEP"); INSERT INTO common_asset_details(asset_id, forked) VALUES("STEP", NULL);
+UPDATE assets SET cryptocompare="MNST" where identifier="MNST";
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("MNST", "S", "MoonStarter.net", "MNST", 1622997981, NULL, "moonstarter", "MNST", "MNST"); INSERT INTO common_asset_details(asset_id, forked) VALUES("MNST", NULL);
+UPDATE assets SET cryptocompare="XAVA" where identifier="XAVA";
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("XAVA", "X", "Avalaunch", "XAVA", 1620038770, NULL, "avalaunch", "XAVA", "XAVA"); INSERT INTO common_asset_details(asset_id, forked) VALUES("XAVA", NULL);
+UPDATE assets SET cryptocompare="ROSN" where identifier="ROSN";
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("ROSN", "S", "Roseon Finance", "ROSN", 1620551645, NULL, "roseon-finance", "ROSN", "ROSN"); INSERT INTO common_asset_details(asset_id, forked) VALUES("ROSN", NULL);
+UPDATE assets SET coingecko="chronologic" where identifier="_ceth_0xE814aeE960a85208C3dB542C53E7D4a6C8D5f60F";
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xE814aeE960a85208C3dB542C53E7D4a6C8D5f60F", 18, ""); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xE814aeE960a85208C3dB542C53E7D4a6C8D5f60F", "C", "Chronologic", "DAY", 1505774047, NULL, "chronologic", "", "0xE814aeE960a85208C3dB542C53E7D4a6C8D5f60F");
+UPDATE assets SET coingecko="sx-network" where identifier="_ceth_0x99fE3B1391503A1bC1788051347A1324bff41452";
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x99fE3B1391503A1bC1788051347A1324bff41452", 18, ""); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x99fE3B1391503A1bC1788051347A1324bff41452", "C", "SportX", "SX", 1610747210, NULL, "sx-network", "", "0x99fE3B1391503A1bC1788051347A1324bff41452");
+UPDATE assets SET coingecko="uniqueone-photo" where identifier="_ceth_0xF6eA0E60cee427B90aA36f327eE61d1d4CABC49F";
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xF6eA0E60cee427B90aA36f327eE61d1d4CABC49F", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xF6eA0E60cee427B90aA36f327eE61d1d4CABC49F", "C", "FOTO", "FOTO", 1612087883, NULL, "unique-photo", "FOTO", "0xF6eA0E60cee427B90aA36f327eE61d1d4CABC49F");
+UPDATE assets SET coingecko="aircoin-2" where identifier="AIR-2";
```

Comment:
How can you be sure it's this coin?

Reply:
In fact after doing a little bit more of research it doesn't seem to be the case. Will just add it to the list of identifiers reviewed for coingecko